target/linux/rtl838x/dts/rtl8380_tp-link_t2500g-10ts.dts

```diff
@@ -0,0 +1,302 @@
+/dts-v1/;
```

Comment:
Please add a SPDX license identifier for all DTS(I) files.

Reply:
Fixed.